UI/Web/src/app/book-reader/_components/quill-wrapper/quill-wrapper.component.html

```diff
@@ -0,0 +1,36 @@
+<div [formGroup]="formGroup()!">
+  <quill-editor class="w-100"
+                [format]="format()"
+                [theme]="theme()"
+                [formControlName]="controlName()"
+                (onContentChanged)="contentChanged.emit($event)">
+
+    <div quill-editor-toolbar>
+
+      @for (toolbarGroup of toolbar(); track $index) {
+
+        <div class="ql-formats">
+          @for (toolbarItem of toolbarGroup; track $index) {
+            <span>
+
+              @if (toolbarItem.value !== undefined) {
+                <button type="button" [class]="toolbarItem.key + ' quil-wrapper-btn'" [value]="toolbarItem.value"></button>
+              } @else if (toolbarItem.values !== undefined) {
+                <select [class]="toolbarItem.key + ' quil-wrapper-select'">
```

Comment:
Not something to fix in this PR, I can do after merge, but some light accessibility would be nice.